Add additional configuration parameters (#55)
ConsenSys/doc.rollups: docs/Reference/Configuration-File.md

    We recommend you use a different ID for each operator instance.

### `vertx_config`

This section contains the [Vert.x](https://vertx.io/docs/) settings.

#### `max_event_loop_execution_time_millis`

> Value of max [event loop execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxEventLoopExecuteTime-long-), in milliseconds
> 
> #### `max_worker_thread_execution_time_millis`
> 
> Value of [max worker execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxWorkerExecuteTime-long-), in milliseconds

Reviewer: ```suggestion
Value of max [event loop execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxEventLoopExecuteTime-long-), in milliseconds.

#### `max_worker_thread_execution_time_millis`

Value of [max worker execute time](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setMaxWorkerExecuteTime-long-), in milliseconds.
```

#### `log_stacktrace_threshold_millis`

Threshold time in milliseconds to trigger a warning containing a [stack trace](https://vertx.io/docs/apidocs/io/vertx/core/VertxOptions.html#setWarningExceptionTime-long-).

#### `http_verticles`

> Number of processing units, for HTTP requests.

Reviewer: ```suggestion
Number of processing units for HTTP requests.
```

(Make consistent with the following section.)

!!! important

>     Do not set higher than the max number of CPUs in the underlying machine.

Reviewer: ```suggestion
    Don't set `http_verticles` higher than the maximum number of CPUs in the underlying machine.
```

#### `websocket_verticles`

> Number of processing units for Websockets requests.

Reviewer: ```suggestion
Number of processing units for WebSocket requests.
```

!!! important

>     Do not set higher than the max number of CPUs in the underlying machine.

Reviewer: ```suggestion
    Don't set `websocket_verticles` higher than the maximum number of CPUs in the underlying machine.
```

WebSocket JSON-RPC listening port to access the manager's APIs.

#### `monitoring_port`

HTTP port for monitoring the following endpoints: `/health`, `/metrics`, and `/live`.

#### `version`

JSON-RPC API version.

### `api.auth`

Details on API authorizations.

#### `rollup_account_max_token_duration`

> Max time a Rollup account session can last. Tokens are issued on login and cannot have a duration time longer than this.

Reviewer: ```suggestion
Maximum time a Rollup account session can last. Tokens are issued on login and can't have a duration time longer than this.
```

#### `api_account_token_duration`

Duration of API accounts tokens.

!!! note

>     API accounts are special accounts that can read the state of all Rollup Accounts. Credentials & Permissions for these accounts are specified in a credentials.toml file.

Reviewer: ```suggestion
    API accounts are special accounts that can read the state of all Rollups accounts. Credentials and permissions for these accounts are specified in a `credentials.toml` file.
```

#### `clock_leniency`

> Amount time desynchronization tolerated between the users and server for JWT token validation.

Reviewer: ```suggestion
Amount of time desynchronization tolerated between the users and server for JWT token validation.
```

Address (host and port) of the engine.

#### `type`

Type options are `Real` and `Fake`. The default is `Real`.

!!! warning

    The `Fake` option is used for testing to start the operator without the engine dependency.

>     This option must not be used in production.

Reviewer 2: Maybe worthy mentioning that it will be deprecated soon in future versions.
